src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
The problem is that `plt.subplots` appears to modify global state, which is exactly what we're trying to avoid by passing around local figures and axes explicitly.  So we shouldn't have *any* calls to `plt.*`, as far as I can tell.